source/includes/trigger-auto-resume.rst

```diff
@@ -0,0 +1,5 @@
+If enabled, when this Trigger's resume token
+cannot be found in the cluster's oplog, the Trigger automatically resumes
+processing events at the next relevant change stream event.
+All change stream events from when the resume token is lost until the Trigger
```

Comment:
Not sure if this is clearer, but is it technically "since the trigger was suspended" rather than "when the resume token was lost"?

Reply:
good catch. rephrasing this to have clearer focus on trigger suspension, not reason behind it of token being lost